querybook/webapp/components/CodeMirrorTooltip/TableTooltip.tsx

```diff
@@ -15,13 +16,15 @@ interface IProps {
     table: IDataTable;
     columns: IDataColumn[];
     schema: IDataSchema;
+    showPinItButton?: boolean;
     openTableModal?: () => any;
 }
 
 export const TableTooltip: React.FunctionComponent<IProps> = ({
     table,
     columns,
     schema,
+    showPinItButton = true,
```

Comment:
just default it to false, since i saw its false below

Reply:
this will be used by other place, like the query editor tooltip, which is true. just to not breaking them without updating them.

Reply:
in this case, we can rename this to hidePinItButton and default to False, and have whatever is calling TableTooltipByName to pass that as True